### PR TITLE
Refactor benchmarks

### PR DESCRIPTION
**Pull request by birkskyum**
Break up the components to individual files, and refactor most from react classes to functional components.

### CODE REVIEW

**Review by wipfli — APPROVED** [review]
I did not look at your source code, but from a user perspective, this seems to do the same as before. Thanks for refactoring!

File: bench/components/StatisticsPlot.tsx [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                        </g>;
+                    })}
+                </g>
+            </svg>
+        );
+    }
+
+    componentDidMount() {
+        // Old API that is not advised anymore
+        this.setState({width: (this as any).ref.clientWidth});
```
Comment (HarelM) [inline bench/components/StatisticsPlot.tsx:208]: Any chance to find a way to remove this "old style" stuff?
I'm not a react expert...

**Comment by HarelM** [comment]
Nice work! This is how it should've been written in the first place :-)

File: bench/components/StatisticsPlot.tsx (hunk 1 above)
Comment (birkskyum) [reply bench/components/StatisticsPlot.tsx:208]: it's a bit more tricky, but i think i can manage

Comment (birkskyum) [reply bench/components/StatisticsPlot.tsx:208]: It's here #909

**Pull request merged**